pkg/config/config.go

```diff
@@ -4,7 +4,13 @@ Copyright IBM Corp. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-// EXPERIMENTAL - Package config allows the creation, retrieval, and modification of channel configtx.
+// EXPERIMENTAL
+//
+// Package config provides utilities to create and modify an channel configuration transaction. Channel transactions contain the configuration data defining members and policies for a system chain (ordering service) or channel.
+// A channel transaction can be a transaction that creates a new channel or modifies a existing channel. Both the creation of a new channel or modification of an existing channel outputs an unsigned transaction represented in a protobuf binary format that must be signed by the requisite number of members such that the
+// transaction fulfills the channel's modification policy.
+//
+// See https://hyperledger-fabric.readthedocs.io/en/master/configtx.html#anatomy-of-a-configuration for a in-depth of channel anatomy.
```

Comment:
`for an in-depth description of a channel configuration's anatomy`

Reply:
🛠️